tests/sample/droppacket.c

```diff
@@ -21,6 +21,7 @@
 #include "net/if_ether.h"
 #include "net/ip.h"
 #include "net/udp.h"
+#include "net_ebpf_ext_xdp_hooks.h"
```

Comment:
Is net_ebpf_ext_xdp_hooks.h included in the ebpf developer nuget? If not, this sample will not compile. IMO sample code should compile.

Reply:
When the v0.20.0 ebpf-for-windows nuget is released (without xdp headers and hooks), we should have the TODO xdp-for-windows nuget with included xdp headers and hooks. 
In addition, the samples are planned to move to xdp-for-windows (not sure of the ETA). Sounds good?